src/shared/components/oncoprint/DeltaUtils.spec.ts

```diff
@@ -513,4 +517,172 @@ describe("Oncoprint DeltaUtils", ()=>{
             assert.equal(oncoprint.setSortConfig.callCount, 0);
         });
     });
+
+    describe('transitionHeatmapTrack() for molecular profile', () => {
```

Comment:
I don't understand this message - "for molecular profile". can you explain more?

Reply:
nevermind, I understand now.